src/pages/ApproveAction/components/ApproveBaseTx.tsx

```diff
@@ -118,7 +119,7 @@ export function BaseTxView({
                       <AvalancheColorIcon size={'32px'} sx={{ mr: 1 }} />
                     )}
                     <Typography variant="h6">
-                      {out.assetDescription?.symbol}
+                      {out.assetDescription?.symbol ?? (out.isAvax && 'AVAX')}
```

Comment:
I think currently this could print `'false'` out.
```suggestion
                      {out.assetDescription?.symbol ?? (out.isAvax ?  'AVAX': '')}
```